code/ai/aicode.cpp

```diff
@@ -1213,10 +1213,9 @@ void ai_turn_towards_vector(vec3d *dest, object *objp, float frametime, float tu
 	}
 
 	//	Dave Andsager: The non-indented lines here are debug code to help you track down the problem in the physics
-	//	that is causing ships to inexplicably rotate very far.  If you hit the Int3(), set the next statement to be
+	//	that is causing ships to inexplicably rotate very far.  If you hit the Warning, set the next statement to be
```

Comment:
Given that it's an `mprintf` (which doesn't halt execution) and not a `Warning`, this comment may need further clarification.